ts-tests/index.ts

```diff
@@ -12,6 +12,7 @@
  */
 
 import { SplitFactory } from '@splitsoftware/splitio';
+import { Attributes, AttributeType } from '../types/splitio';
```

Comment:
Emma, I think you can remove this line and replace the occurrences of `Attributes` with `SplitIO.Attributes` in the code. Same for `AttributeType`